tools/release-prepare.sh

```diff
@@ -66,16 +66,18 @@ for file in "${files_to_change[@]}"; do
     sed -i "s/$prev_ver/$version/" "$file"
 done
 
-# Run `cargo check` to update firecracker and jailer versions in
-# `Cargo.lock`.
-say "Updating lockfile..."
+CHANGED=()
 cargo check
-CHANGED=(Cargo.lock)
 
-cd tests/integration_tests/security/demo_seccomp
-cargo check
-cd -
-CHANGED+=(tests/integration_tests/security/demo_seccomp/Cargo.lock)
+# Run `cargo check` to update firecracker and jailer versions in all
+# `Cargo.lock`.
+for cargo_lock in $(find . -path ./build -prune -o -name Cargo.lock -print); do
```

Comment:
This breaks if a Cargo.lock file is ever found on a path containing a space (e.g. if you create the file "little test/Cargo.lock") the output will be (I think using `-exec` instead would work here)

```bash
$ for cargo_lock in $(find . -path ./build -prune -o -name Cargo.lock -print); do echo $cargo_lock; done;
./tests/host_tools/uffd/Cargo.lock
./tests/integration_tests/security/demo_seccomp/Cargo.lock
./Cargo.lock
./little
test/Cargo.lock
```

Reply:
How about prohibiting creation of spaced files/directories in the repo? :)

Reply:
The advantage of not dealing with this would be that __it will break__ during release preparation, at which point we can choose to either fix the offending path, or implement this fix.

Reply:
I will make a change to add a comment on this.

Reply:
Yeah, I think we could add a quick check for that. I can add it in this PR.